mail-mta/protonmail-bridge-bin/protonmail-bridge-bin-1.0.4.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit rpm
+
+DESCRIPTION="Serves ProtonMail to IMAP/SMTP clients"
+HOMEPAGE="https://protonmail.com/bridge/"
+SRC_URI="https://protonmail.com/download/${P/-bin/}-1.x86_64.rpm"
+
+RESTRICT="mirror"
+
+LICENSE="MIT protonmail-bridge-EULA"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+DEPEND=""
+RDEPEND="
+	app-crypt/libsecret
+	dev-libs/glib
+	media-libs/mesa
+	media-sound/pulseaudio
+	virtual/opengl
```

Comment:
Do I really wanna know why a mail bridge needs OpenGL and sound system?

Reply:
the gui and notifications, of course :)